lib/modules/platform/bitbucket-server/schema.ts

```diff
@@ -1,8 +1,10 @@
 import { z } from 'zod';
 
+const Email = z.string().email();
+
 export const User = z.object({
   displayName: z.string(),
-  emailAddress: z.string(),
+  emailAddress: Email.optional(),
```

Comment:
```suggestion
  emailAddress: Email.optional().catch(null),
```

if email is an empty string

Reply:
I don't understand fully what you what to accomplish with the `catch` part. Can you explain what is the goal?

Reply:
Ah I see you've made another PR.